data_reports/119_OakRidgeNationalLab_TDE.Rmd

```diff
@@ -1,30 +1,194 @@
----
-title: "119_OakRidgeNationalLab_TDE"
-author: "Julie Bielecki"
-date: "3/28/2021"
-output: html_document
----
+#Oak Ridge National Lab_TDE
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+The Oak Ridge National Lab_TDE data set in ISCN3 contains 1176 layer-level rows and 294 profile-level information rows after cleaning for ISCN3.5.
+Rows weren't removed from ISCN3 because of reasons.
```

Comment:
Say what these reasons were!

Reply:
I didn't notice this line while doing a final look-through, but it was supposed to be deleted a while ago. Removed this line and kept the line that explained the reasoning behind row removal.